tests/integration/ha_tests/manifests/chaos_network_loss.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: chaos-mesh.org/v1alpha1
+kind: NetworkChaos
+metadata:
+  name: network-loss-primary
+  namespace: $namespace
+spec:
+  action: loss
+  mode: one
+  selector:
+    pods:
+      $namespace:
+        - $pod
+  loss:
+    loss: "100"
+    correlation: "100"
+  duration: "60m"
```

Comment:
Taken from the [mysql-k8s charm](https://github.com/canonical/mysql-k8s-operator/pull/116).